test/widgets/content_test.dart

```diff
@@ -1005,6 +1005,65 @@ void main() {
     });
   });
 
+  group('WebsitePreview', () {
+    Future<void> prepare(WidgetTester tester, String html) async {
+      await prepareContent(tester, plainContent(html),
+        wrapWithPerAccountStoreWidget: true);
+    }
+
+    testWidgets('smoke', (tester) async {
+      final url = Uri.parse(ContentExample.websitePreviewSmoke.markdown!);
+      await prepare(tester, ContentExample.websitePreviewSmoke.html);
+      tester.widget(find.byType(WebsitePreview));
+
+      await tester.tap(find.text('Zulip — organized team chat'));
```

Comment:
One more quick check we can add to the two of these that have a description: check `find.text` finds something with the description's text.

(Otherwise e.g. this test case and the `websitePreviewWithoutDescription` one below are making exactly the same set of checks, I think, which seems incomplete since there are expected differences in behavior.)